java-aws-lambda-gradle/build.gradle

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id 'java'
+}
+
+sourceCompatibility = JavaVersion.VERSION_11
+targetCompatibility = JavaVersion.VERSION_11
+
```

Comment:
Since you're setting 11 as the version, please also add something regarding the allowAttachSelf